CONTRIBUTING.md

```diff
@@ -13,9 +13,16 @@ Communication
   * If you are looking for something to work on, ping the Core team using !core <message>.
 * For questions and discussions related to the project, new issues may be opened (using the appropriate labels).
 
+Branch Naming
+-------------
+* For features: feature/<feature_name>
+* For documentation: documentation/<document_name>
+* For bug fixes: hotfix/<bug_name>
```

Comment:
I think 'bug/' implies that you are trying to ask for a 'bug' to be merged into the code as opposed to a 'fix'.

The 'hot' in 'hotfix' implies that the code is intended to be merged live into the master while the app is still running the last release. That isn't really the case here since we're not following a branching release model. So I think the 'hot' can be dropped.

**Suggestions:**
- fix/
- bugfix/
- patch/